cmd/gateway/azure/gateway-azure.go

```diff
@@ -125,6 +131,19 @@ func azureGatewayMain(ctx *cli.Context) {
 	minio.StartGateway(ctx, &Azure{host})
 }
 
+func getUploadChunkSizeFromEnv(envvar string, defaultValue int) int {
+	envChunkSize := os.Getenv(envvar)
```

Comment:
you can use `env` package at `pkg/env` to do `env.Get(envvar, defaultValue)`

Reply:
Yes, I looked into that but that would require casting the integer back and forth to float.

I added an extra check to see if the environmental variable is empty. Do you mind if I use it like this?

Reply:
Just use env.Get because it is meant for this purpose, strconv.ParseFloat is quite inexpensive, whatever you want to calculate can be calculated afterwards the parsing is complete..